tests/of-tests/openflow.py.in

```diff
@@ -420,61 +417,6 @@ def runTest(self):
         self.controller.message_send(req)
         do_barrier(self.controller)
 
-class TableStatsGet(openflow_base_tests.OFTestInterface):
```

Comment:
I'm getting rid of this test since global table counters are not supported in bmv2.